modules/ui/map3d_viewer.js

```diff
@@ -243,6 +244,7 @@ export function uiMap3dViewer(context) {
           .on('end', () =>
             selection.selectAll('.three-d-map').style('display', 'none')
           );
+        urlhash.setParam('3d', null);
```

Comment:
After discussing with Bryan this morning, I agree we should rename this param to `map3d`. Since we alphabetize the params, it makes sense for the `map` and `map3d` to be next to one another.

Reply:
Done